### DISCUSSION TITLE
Could this be used to extract tabular data from photos of supplement bottles?

### DISCUSSION THREAD

**Discussion by lilouartz** [mindee/doctr#1556] [General]
![0108_nowproductlabels](https://github.com/mindee/doctr/assets/167234340/5d82a71d-33b8-48f9-97e5-b02eae55ecaf)

My dataset are photos such as this one.

My use case is to extract the "Supplement Facts" table.

Would doctr be a good choice for such a use case?

**Comment by felixdittrich92** [comment]
Hi @lilouartz :wave:,

for pure OCR docTR works well as you can see: (tested with `fast_base`pytorch on main branch)
![Screenshot from 2024-04-17 11-03-13](https://github.com/mindee/doctr/assets/23098070/6ebb391a-a667-4e1d-8230-06ef57178e92)

We don't have any logic yet to extract only tabular data but you could try to combine: https://huggingface.co/docs/transformers/model_doc/table-transformer with docTR for OCR or if the table is on a fixed position work with the predictor hooks (https://mindee.github.io/doctr/using_doctr/using_models.html#advanced-options) to limit the area 

Hope this helps :)

Best regards,
Felix

**Reply by aditya-ihx** [reply]
Are there any ideas or future plans to extract tabular data ?
